test/editor/Store/MenuStoreConnectorTest.re

```diff
@@ -0,0 +1,89 @@
+module MenuStore = Oni_Store__MenuStoreConnector;
+module Model = Oni_Model;
+open TestFramework;
+
+let cmd: Model.Actions.menuCommand = {
+  category: None,
+  name: "command",
+  command: () => Model.Actions.Init,
+  icon: None,
+};
+
+let initial_state = Model.State.create();
+let get_state = menu => {...initial_state, menu};
+
+let commands = [cmd, cmd, cmd, cmd];
+
+let onStateChange = MenuStore.start() |> fst |> Helpers.onStateChange;
+
+let store =
+  describe("MenuStoreConnector", ({describe, _}) => {
+    describe("MenuPreviousItem", ({test, _}) => {
+      test("goes from 1st to last", ({expect}) =>
+        onStateChange(
```

Comment:
Impressive work on these tests - thank you for getting `Store` tests started for us 😍